software/ompi/v5.0/timeline-graph.php

```diff
@@ -0,0 +1,35 @@
+<?php
+$topdir = "../../..";
+include_once("$topdir/includes/timeline-graph.inc");
+
+$vpos = 0;
+$graph = 0;
+$data = array();
+
+# Oldest milestone date
+$first_date = "2021-03-11";
+
+# The graph needs about a year time span for the title to display properly.
+# So pick either a year beyond the first date or 90 days beyond now,
+# whichever is greater.
+
+# 3 months from today
+$ts1 = time() + (90 * 24 * 60 * 60);
+# 1 year beyond the first date (this i
+$ts2 = mktime(0, 0, 0, 6, 1, 2016);
```

Comment:
If you see the timelines on the website, I suspect they're different widths partially because folks have forgotten to update the $ts2.  Reading the code I think it's expected that $ts2 would encode $first_date + 1 year... and then as we keep adding minor releases $ts1 would protect us...  Not sure why the original coder didn't code it as a function of $first_date then.  Anway, please see if you agree that line 19 should be:
`$ts2 = mktime(0, 0, 0, 3, 11, 2021)` ?

Reply:
would it be 2022 not 2021?

Reply:
I pushed up 2021, but lmk if it should be 2022

Reply:
I think you're right also sorry.